cgs_vmc/training.py

```diff
@@ -254,7 +259,7 @@ def build_opt_ops(
     psi_target = target_wavefunction(configs)
 
     loss = tf.reduce_mean(
-        tf.squared_difference(psi, psi_target * np.sqrt(2 ** n_sites)))
+        tf.squared_difference(psi, psi_target * hparams.scale))
```

Comment:
I think here you also want to move the the scaling up, so that `psi_target` has the scale you want.